packages/vue/src/examples/pages/product/Product.vue

```diff
@@ -754,7 +754,7 @@ export default {
   color: $c-text-primary;
   font-family: $body-font-family-secondary;
   font-size: $font-size-regular-mobile;
-  font-width: $body-font-weight-secondary;
+  // font-width: $body-font-weight-secondary;
```

Comment:
```suggestion
  font-weight: $body-font-weight-secondary;
```

Reply:
This one was commented out because I could not run storybook due to problems with this property. 
And that push was made only to prepare work at home not as finished task, so it's obviously not ready. Sorry for confusion.